.vib/kube-prometheus/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/kube-prometheus"
         },
-        "runtime_parameters": "Im9wZXJhdG9yIjoKICAiY3VzdG9tUmVzb3VyY2VEZWxldGVQb2xpY3kiOiAiYmVmb3JlLWhvb2stY3JlYXRpb24iCiJwcm9tZXRoZXVzIjoKICAic2VydmljZSI6CiAgICAidHlwZSI6ICJMb2FkQmFsYW5jZXIiCiAgICAicG9ydHMiOiAKICAgICAgImh0dHAiOiA4MAo=",
+        "runtime_parameters": "b3BlcmF0b3I6CiAgc2VydmljZUFjY291bnQ6CiAgICBjcmVhdGU6IHRydWUKICAgIGF1dG9tb3VudFNlcnZpY2VBY2NvdW50VG9rZW46IHRydWUKICBwb2RTZWN1cml0eUNvbnRleHQ6CiAgICBlbmFibGVkOiB0cnVlCiAgICBydW5Bc1VzZXI6IDEwMDIKICAgIGZzR3JvdXA6IDEwMDIKICBrdWJlbGV0U2VydmljZToKICAgIGVuYWJsZWQ6IHRydWUKICBzZXJ2aWNlOgogICAgcG9ydHM6CiAgICAgIGh0dHA6IDgwODEKICBjdXN0b21SZXNvdXJjZURlbGV0ZVBvbGljeTogYmVmb3JlLWhvb2stY3JlYXRpb24KcHJvbWV0aGV1czoKICBlbmFibGVkOiB0cnVlCiAgcmVwbGljYUNvdW50OiAyCiAgc2VydmljZToKICAgIHR5cGU6IExvYWRCYWxhbmNlcgogICAgcG9ydHM6IAogICAgICBodHRwOiA4MAphbGVydG1hbmFnZXI6CiAgZW5hYmxlZDogdHJ1ZQogIHJlcGxpY2FDb3VudDogMgogIHNlcnZpY2U6CiAgICBwb3J0czoKICAgICAgaHR0cDogOTA5NApleHBvcnRlcnM6CiAgbm9kZS1leHBvcnRlcjoKICAgIGVuYWJsZWQ6IHRydWUKICBrdWJlLXN0YXRlLW1ldHJpY3M6CiAgICBlbmFibGVkOiB0cnVlCiAgICByZXBsaWNhQ291bnQ6IDEKa3ViZWxldDoKICBlbmFibGVkOiB0cnVl",
```

Comment:
There are couple of things about the `runtime_parameters` I've noticed:

- You are missing `operator.enabled=true`
- You should set `X.serviceMonitor.enabled=true` to ensure that metrics from the different components checked in Cypress are scraped correctly.
- There is no current entry for `operator.customResourceDeletePolicy` in the values.yaml, so I believe you can safely remove this (even if it was set before 😁 )
- I could not see an entry for `exporters.kube-state-metrics.replicaCount`, I assume you can also safely remove it.

As additional advice (however, no strong thoughts on this), I think you can also set `containerSecurityContext.capabilities.drop:[ALL]` and check this on GOSS. I did something similar in the [MetalLB tests](https://github.com/bitnami/charts/pull/12992/files#diff-a0aec40c162050a65bb2687bc4ef6c7e4b8923cb6bd9e7c7e55ab32532622661R11).